index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title>Waseem Salem Bio</title>
+  </head>
+  <body>
+    <h1>Waseem Salem</h1>
+    <img src="images/waseem_salem_img.jpg" alt="Waseem Salem">
+    <p>Talented Art Director with over 15 years of experience in graphic design,
+      advertising and marketing. Experience includes the development of creative
+      and advertising campaigns in both creative and management roles.
+      Possesses technical skills, art direction, photography skills, and printing expertise.
+      <br><br>
+      Proficiency in Adobe Photoshop, Illustrator, InDesign, Dreamweaver, and MS office, in both Mac and PC platforms.
+      Knowledge in web design (HTML, CSS).</p>
+<br>
+      <h3>Some of my interestes</h3>
+      <p>I have so many dreams in my personal & my proffesional lives, on the professional side My dream is
+         to become a web developer and apply my creativity on website and mobile applications.
+         Also, to keep learning and growing in my carrer
+         <br>
+
+         As well, one of my dreams to travel to Rio De Janeiro in Brazil, and learn about the country and it's culture
+      </p>
+      <br>
+      <h3>Some of my interestes</h3>
+      <ul>
+        <li>Art</li>
+        <li>Video Games</li>
+        <li>Basketball</li>
+        <li>Concerts</li>
+      </ul>
+<br>
+<br>
+<h2>Brazil</h2>
+<!--Flag_of_Brazil-->
+  <h4>Flag of Brazil</h4>
```

Comment:
For this section where you have sections of headings and images, you can considerusing paragraphs to divide the sections instead of using two `<br>` tags.

E.g.
``` html
<p>
  <h4>Flag of Brazil</h4>
  <img src="https://upload.wikimedia.org/wikipedia/en/thumb/0/05/Flag_of_Brazil.svg/1200px-Flag_of_Brazil.svg.png" alt="Flag of Brazil" width="400px" height="auto">
</p>
<p>
  <h4>Rio De Janeiro</h4>
  <img src="https://www.travelinglifestyle.net/wp-content/uploads/2018/01/best-hostels-in-rio-de-janeiro-brazil-759x500.jpg" alt="Rio De Janeiro" width="400px" height="auto">
</p>
<p>
  <h4>Best beaches in teh world</h4>
  <img src="https://www.riodejaneiro.com/media/uploads/riodejaneiro.com/2014/09/copacabana-beach.jpg"
    alt="Rio De Janeiro beach" width="400px" height="auto">
</p>
```